src/lib.rs

```diff
@@ -126,12 +108,15 @@ pub struct OrgList {
     pub orgs: Vec<Org>,
 }
 
+#[serde_as]
```

Comment:
Does this annotation do anything if there is nothing defined on individual fields?

Reply:
good catch, this is legacy from when I was returning solana pubkeys